github_nodes_test.go

```diff
@@ -151,7 +152,10 @@ func TestGithubNodes(t *testing.T) {
 
 			h := new(GithubNodeProcessor)
 
-			err = h.InitWith(db, cli, topic)
+			ghdb, err := github.OpenDatabase(addr, 1, 1)
+			require.NoError(t, err)
```

Comment:
```suggestion
			require.NoError(t, err)
			defer ghdb.Close()
```
Same in other places.

Reply:
Done.